enos/modules/test_e2e/main.tf

```diff
@@ -139,7 +144,7 @@ resource "enos_local_exec" "run_e2e_test" {
     E2E_AWS_HOST_SET_IPS2         = local.aws_host_set_ips2
   }
 
-  inline = ["set -o pipefail; PATH=\"${var.local_boundary_dir}:$PATH\" go test -v ${var.test_package} -count=1 -json | tparse -follow -format plain 2>&1 | tee ${path.module}/../../test-e2e-${local.package_name}.out"]
+  inline = var.debug ? ["pwd"] : ["set -o pipefail; PATH=\"${var.local_boundary_dir}:$PATH\" go test -v ${var.test_package} -count=1 -json | tparse -follow -format plain 2>&1 | tee ${path.module}/../../test-e2e-${local.package_name}.log"]
```

Comment:
you could also do this with a `count` block (count of 0 if true, otherwise 1) the only downside is the count gets added to the end of all your resources (something like `enos_local_exec.run_e2e_test[0]`)

Reply:
When doing some testing, it looks like when `count=0`, it doesn't set up that resource at all, so it doesn't show up in the terraform state list. Because of that, I can't get the environment variables out from the script.